kong/tracing/instrumentation.lua

```diff
@@ -151,8 +152,8 @@ function _M.http_client()
       attributes["http.proxy"] = http_proxy
     end
 
-    local span = tracer.start_span("HTTP " .. method .. " " .. uri, {
-      span_kind = 3,
+    local span = tracer.start_span("kong.internal.request", {
```

Comment:
What is an internal request?

Reply:
any request that uses `resty.http`. These can be triggered as part of a plugin's execution (oidc), or because of custom logic defined in serverless functions, etc.
As mentioned above, we are looking for suggestions to finalize the choice of all these names.

Reply:
Ah, that looks like an 'external' request to me. :smile: 
This is tricky and it is breaking my brain!

Reply:
I know, I had the exact same mental conflict when I started thinking about it :laughing: